src/txkube/test/test_authentication.py

```diff
@@ -171,13 +171,13 @@ def test_http_bearer_token_authorization(self):
         issues.  The header includes the bearer token from the service account
         file.  This works over HTTP.
         """
-        token = bytes(uuid4())
+        token = str(uuid4()).encode("ascii")
```

Comment:
`str` returns `bytes` on Python 2 ... which does not need to be encoded (misfeature that it is encodable at all).